src/Events/Editor/Full_Site/Assets.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace TEC\Events\Editor\Full_Site;
+
+use \Tribe__Events__Main as Events_Main;
+
+/**
+ * Class Assets
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Editor\Full_Site
+ */
+class Assets extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * Binds and sets up implementations.
+	 *
+    * @since TBD
+	 */
+	public function register() {
```

Comment:
I'm not a fan of anemic service providers; they can be broken out later, if they require it.

Reply:
If this was left in place by choice, could you provide some context for the choice?

Reply:
I've removed it.